tests/test_property.py

```diff
@@ -508,14 +509,14 @@ def test_remove_part_from_dict(example_model_with_properties):
     model = example_model_with_properties
     pc = model.grid().property_collection
     assert pc is not None
-    assert len(pc.parts()) == 7
+    assert len(pc.parts()) == 8
```

Comment:
Is this an expected change in behaviour in the code?

Reply:
8 is the correct expectation here: the example_model_with_properties builds 7 explicit grid properties but the grid included in that model has the (default) addition of an active cell property. I don't think the resqpy code has changed in that regard for a very long time.